hw_8_new.py

```diff
@@ -0,0 +1,57 @@
+# Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+# Декоратор має працювати для різних функцій однаково.
+import time
+def my_decorator(func):
+    def deco_func():
+        print(time.strftime("%H:%M:%S"))
+        func()
+        print(func.__name__)
+    return deco_func()
+
+@my_decorator
+def my_func():
+    pass
+
+# Написати кастомний Exception клас, MyCustomException, який має повідомляти "Custom exception is occured".
+class MyCustomException(Exception):
+    pass
+try:
+  raise MyCustomException('Custom exception is occured')
+except Exception as error:
+  print(error)
+
+# Написати власний менеджер контексту, задачею якого буде друкувати "==========" – 10 знаків дорівнює
+# перед виконанням коду та після виконання коду, таким чином виділяючи блок коду символами дорівнює.
+# У випадку виникнення будь-якої помилки вона має бути надрукована текстом, проте програма не має завершити своєї роботи.
+class CtxManager:
+    def __init__(self,value):
+        self.value=value
+    def __enter__(self):
+        print('==========')
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print('==========')
```

Comment:
You should check here if there is some exception (`if exc_type:`) and print its value. To avoid raising the exception add `return True` at the end of the method